pkg/volume/aws_ebs/aws_ebs.go

```diff
@@ -421,6 +421,17 @@ type awsElasticBlockStoreProvisioner struct {
 var _ volume.Provisioner = &awsElasticBlockStoreProvisioner{}
 
 func (c *awsElasticBlockStoreProvisioner) Provision() (*v1.PersistentVolume, error) {
+	var FSType string = "ext4"
+
+	for k, v := range c.options.Parameters {
+		switch strings.ToLower(k) {
+		case volume.VolumeParameterFSType:
+			FSType = v
```

Comment:
I feel like _somewhere_ we might want to sanity check these, because I think this will be the equivalent of a SQL injection attack.  Not sure where yet, but this is probably as good a place as any.